components/cam/bld/namelist_files/namelist_definition.xml

```diff
@@ -6067,27 +6081,28 @@ Constrained density reconstuctor (CDR) for property preservation:
 20: QLT with superlevels of 8 levels.
 21: QLT with superlevels of 8 levels; use local CAAS over sublevels.
 30: CAAS with superlevels of 8 levels.
-Default: 20
+Default: 20 (set by dycore)
```

Comment:
We have settled on 21 rather than 20, so we should use 21 here.

Reply:
thanks - I think best would be to not give a default here, if the default value is set in the dycore.  So I'll change all of these to:
Default: (set by dycore)